action.yml

```diff
@@ -83,7 +83,27 @@ runs:
             process.exit(1);
           }
 
+    - name: Set Condition Variables
+      id: condition-vars
```

Comment:
if you don't use the `id` later, there is no need to specify it (relevant for all steps below)